Hamburg/HLC/RDW005.xml

```diff
@@ -0,0 +1,169 @@
+                           <dimensions type="outer" unit="mm" xml:id="strip1">
+                              <height>763</height>
+                              <width>45</width>
+                           </dimensions>
+                           <dimensions type="outer" unit="mm" xml:id="strip2">
+                              <height>450</height>
+                              <width>45</width>
+                           </dimensions>
+                        </extent>
+                        <!--<condition key=""></condition>-->
+                     </supportDesc>
+                     
+                        <layoutDesc>
+                           <layout columns="1" writtenLines="">
+                              <note>Number of characters per line: </note>
+                                 <dimensions unit="mm" xml:id="textarea1">
+                              <height>425</height>
+                              <width>39</width>
+                           </dimensions>
+                           </layout>
+                           
+                           
+                           <layout columns="1" writtenLines="">
+                              <note>Number of characters per line: </note>
+                              <dimensions unit="mm" xml:id="textarea2">
+                                 <height>510</height>
+                                 <width>40</width>
+                              </dimensions>
+                           </layout>
+                        </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--    <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                            <date notBefore="" notAfter=""/>
+                           <desc></desc>
+                           <seg type="ink"></seg>
+                           <seg type="script"></seg>
+                           <seg type="rubrication"></seg>
+                        </handNote>
+                     </handDesc>-->
+                  
+                  <decoDesc>
+                     <decoNote xml:id="d1" type="drawing">
+                        <desc></desc>
+                        <dimensions unit="mm">
+                           <height>173</height>
+                           <width>44</width>
+                        </dimensions>
+                     </decoNote>
+                     
+                     <decoNote xml:id="d2" type="drawing">
+                        <desc></desc>
+                        <dimensions>
+                           <height>75</height>
+                           <width>46</width>
+                        </dimensions>
+                     </decoNote>
+                  </decoDesc>
+
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <decoNote xml:id="b1" type="SlipCase">The scroll is kept in a case probably made of reptile skin.</decoNote>
```

Comment:
Perhaps a more specific term can be used to refer to the traditional container (?) used for keeping Ethiopic scrolls. Is it European- or Ethiopian-made?